### PR TITLE
Update profanity.py

### PR DESCRIPTION
**Pull request by n00bility**
A much simpler yet more powerful method for censoring inappropriate words from messages compared to the previous one.

### CODE REVIEW

File: dist/ba_root/mods/features/profanity.py [hunk 1]
```diff
@@ -1,60 +1,47 @@
...
+# Simple yet powerful tools to censor bad words from chat messages
+# It doesn't use regex but string comparison, so hopefully it will be faster
+# With a built-in long list of profanity words (in English), there’s no need to define it yourself
+# You can add your custom words for better coverage
+# The censor can be bypassed by adding any character(s) to the words, unfortunately :(
+# But hey, at least it can attempt to censor bad words written in fancy letters :)
+# Don’t forget to install the unidecode and better-profanity modules
+# Brought to you by goblogger
+
+from better_profanity import profanity
```
Comment (imayushsaini) [inline dist/ba_root/mods/features/profanity.py]: where better_profanity module is defined?

Comment (vishal332008) [reply dist/ba_root/mods/features/profanity.py]: it's a package. I guess it should be downloaded to be used.

**Comment by n00bility** [comment]
Yes I mean they're packages (better-profanity and unidecode).
We need to install them first using pip.

**Comment by imayushsaini** [comment]
> Yes I mean they're packages (better-profanity and unidecode). We need to install them first using pip.
> […](#)

Cant we bundle it here only ? in dist/ba_data/python-site-packages ?
Folks can face issues setting up a server the first time.

**Pull request closed without merging**